kratos/processes/split_internal_interfaces_process.cpp

```diff
@@ -72,24 +88,30 @@ void SplitInternalInterfacesProcess::SplitBoundary(const std::size_t PropertyIdB
     max_node_id++;
 
     std::map<std::size_t, Node<3>::Pointer> new_nodes_map;
```

Comment:
why not unordered?

Reply:
Not 100% sure as this was already there, but I think that for the proper construction of the interface geometries we need to keep the insertion order, which is done base in the original nodes one.